packages/dm-core-plugins/src/generic-list/GenericListPlugin.tsx

```diff
@@ -64,6 +67,8 @@ export const GenericListPlugin = (
   const [itemsExpanded, setItemsExpanded] = useState<{
     [key: string]: boolean
   }>({})
+  const [dirtyOrder, setDirtyOrder] = useState<boolean>(false)
```

Comment:
I was a bit confused about the `dirtyOrder` variable name

Maybe something like "areItemsReordered" is better.

Reply:
renamed to "unsavedChanges"